resources/views/test-view-1.blade.php

```diff
@@ -0,0 +1 @@
+<?= echo'<br> This is test view 1';
```

Comment:
В Blade файлах не нужно делать echo, это обычный HTML, но с возможностью использовать PHP.
```suggestion
<br> This is test view 1
```

Reply:
> В Blade файлах не нужно делать echo, это обычный HTML, но с возможностью использовать PHP.

А может быть лучше вообще эти test_view уже удалить?
я просто думал, что в первом дз надо самому создать какие-то новые view, а не просто создавать такой же код, как и был на лекции...